internal/kubernetes/test/nodeTestCases.go

```diff
@@ -11,6 +11,50 @@ var (
 	Cpu2, _    = resource.ParseQuantity("2")
 	Memory1, _ = resource.ParseQuantity("1000")
 	Memory2, _ = resource.ParseQuantity("2000")
+	Condition1 = []corev1.NodeCondition{
```

Comment:
Since we address this variable also in virtual kubelet code, I would move it to another file (not testing)

https://github.com/liqotech/liqo/blob/ce42ff0d6b739d034dbccbd23c086746d83a557e/internal/kubernetes/virtualNodeupdater.go#L193

Reply:
I agree with you, I reused that variable just to avoid some lines of code but it's better not to use test files in the normal code